extension/js/common/core/common.ts

```diff
@@ -362,4 +362,8 @@ export const asyncSome = async<T>(arr: Array<T>, predicate: (e: T) => Promise<bo
     if (await predicate(e)) return true;
   }
   return false;
+};
+
+export const stringTuple = <T extends [string] | string[]>(...data: T): T => {
```

Comment:
Wouldn't `<T extends string[]>` do?
I see no benefit of `| [string]`

Reply:
You are right. 👍 
Just updated